src/libraries/sendtohelixhelp.proj

```diff
@@ -169,7 +169,7 @@
 
       <_WorkItem Include="$(WorkItemArchiveWildCard)" Exclude="$(HelixCorrelationPayload)" />
 
-      <HelixWorkItem Include="@(_WorkItem -> '%(FileName)')">
+      <HelixWorkItem Include="@(_WorkItem -> '%(FileName)')" Condition="'@(_WorkItem->Count())' != 0">
```

Comment:
```suggestion
      <HelixWorkItem Include="@(_WorkItem -> '%(FileName)')" Condition="'@(_WorkItem)' != ''">
```

Reply:
curious, why is this necessary? is msbuild not just ignoring the item if it's empty?

Reply:
no it still adds ```F:\workspace\_work\1\s\artifacts\helix/runtime/packages-testPayload-Debug.zip```